roles/update-host/tasks/main.yml

```diff
@@ -1,5 +1,6 @@
 ---
 
 - import_tasks: update-host.yml
-- import_tasks: reboot-host.yml
-- import_tasks: wait-for-host.yml 
+  when:
```

Comment:
Please keep the conditional in the `update-host` file (and see overall comments about keeping reboot-host and wait-for-host)